### PR TITLE
Refactor: Add Actions heading to button column in all the tables

### LINKED ISSUE #878: Refactor: Add/replace 'Actions' heading to buttons column in all tables
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**
Many tables either have an empty heading or 'Action' for the column containing buttons 

**Expected behavior:**
The heading for the column containing buttons should be 'Actions' in all the tables

### PR DESCRIPTION
**Pull request by ritikamotwani**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Refactor:  Add  'Actions' heading to buttons column in all tables.

Fixes #878

### CODE REVIEW

**Comment by niranjan94** [comment]
Deployment link: https://pr-879-open-event-frontend-fossasia.surge.sh

File: app/templates/events/view/speakers/list.hbs [hunk 1]
```diff
@@ -7,7 +7,7 @@
         <th>{{t 'Phone'}}</th>
         <th>{{t 'Submitted Sessions'}}</th>
         <th>{{t 'State'}}</th>
-        <th></th>
+        <th>{{t 'Actions'}}</th>
```
Comment (geekyd) [inline app/templates/events/view/speakers/list.hbs]: Please revert this, We will be using ember models table here

**Comment by niranjan94** [comment]
Deployment link: https://pr-879-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-879-open-event-frontend-fossasia.surge.sh

**Comment by ritikamotwani** [comment]
@geekyd please review!

**Pull request merged**